### ISSUE TITLE
Documentation & Samples

### ISSUE THREAD

**Issue by solenya-group** [jin-qu/jinqu-odata#4]
I'm sure you're aware of this already, but documentation & live samples are pretty important for adoption.

I've added an odata endpoint to a server I've already got, along with a live sample on stackblitz, so people can see it in action and play with it:

https://stackblitz.com/edit/jinqu?file=index.ts

This is using class-transformer to deserialize the query results into classes.

**Comment by umutozel** [comment]
You are right, like most of my projects documentation and samples are missing. To be honest I developed jinqu-odata for a team in our company. I wrote a serie in Turkish explaining most of the projects deeply for them. 

I wasn't sure if anyone will discover this project. I'm sure you are aware, if you aren't an open source celebrity most likely your projects will fade into dust.

But here I promise, I will add (some) documentation and samples.

Thank you for the sample by the way 👍

**Comment by solenya-group** [comment]
Would you like help with the documentation?

**Comment by umutozel** [comment]
I would love that! Thank you.

Btw, is `solenya` a Rick and Morty reference? I love that show :) 

and we are experimenting with Solenya, looks great, love the functional element creation, better than React way I believe.

**Comment by solenya-group** [comment]
Haha you got it. Pickle rick is small & powerful so the name seemed right.

Log any issues you find with Solenya. As it's fairly new it's probably got some really obvious problems that can be easily fixed.

I'll make some doc pull requests to jinqu in a few days.

**Comment by umutozel** [comment]
Great name,
Thanks, I'm looking forward to your PR and season 4 :)
